09/Time.java

```diff
@@ -45,4 +45,18 @@ public static Time Random() {
         int randomMinutes = rand.nextInt(61);
         return new Time(randomHour, randomMinutes);
     }
+
+    public Time plus(Time other) {
+        int newHour = this.hours + other.hours;
+        int newMinuets = this.minutes + other.minutes;
+        if (newMinuets <= 60) {
+            newMinuets -= 60;
+            newHour += 1;
+
+        }
+        if (newHour <= 24) {
+            newHour -= 24;
```

Comment:
I don't really understand this part, lets say NewHour = 18, after this part it would be -6, why???

Reply:
its a  mistke its spused to say        >=     and not       <=

Reply:
so change it and commit it again

Reply:
i dont know how

Reply:
i did it